lab_6_pipeline/pipeline.py

```diff
@@ -144,16 +229,89 @@ def __init__(self, corpus_manager: CorpusManager):
         """
         Initializes MorphologicalAnalysisPipeline
         """
+        self._corpus = corpus_manager
+        mapping_dir = Path(__file__).parent / 'data'
+        if not mapping_dir.exists():
+            mapping_dir.mkdir(parents=True)
+        mapping_file = mapping_dir / 'mystem_tags_mapping.json'
+        if not mapping_file.exists():
+            mapping_file.touch()
+        if not mapping_file.stat().st_size:
+            ud_mapping = {'POS': {'S': 'NOUN',
+                                  'SPRO': 'PRON',
+                                  'A': 'ADJ',
+                                  'ANUM': 'ADJ',
+                                  'APRO': 'ADJ',
+                                  'COM': 'ADJ',
+                                  'V': 'VERB',
+                                  'NUM': 'NUM',
+                                  'ADV': 'ADV',
+                                  'ADVPRO': 'ADV',
+                                  'PR': 'ADP',
+                                  'CONJ': 'CCONJ',
+                                  'PART': 'PART',
+                                  'INTJ': 'INTJ'}}
+            with open(mapping_file, 'w', encoding='utf-8') as json_file:
```

Comment:
json file should be defined statically (i.e. create file in advance and read it on premise)

Reply:
Done